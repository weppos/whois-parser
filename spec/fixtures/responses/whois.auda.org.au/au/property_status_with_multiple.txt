Domain Name: PINEWOOD.COM.AU
Registry Domain ID: D407400000001847702-AU
Registrar WHOIS Server: whois.synergywholesale.com
Registrar URL: whois.synergywholesale.com
Last Modified:
Registrar Name: Synergy Wholesale Pty Ltd
Registrar Abuse Contact Email: abuse@synergywholesale.com
Registrar Abuse Contact Phone: +61.383999483
Reseller Name:
Status: serverRenewProhibited https://afilias.com.au/get-au/whois-status-codes#serverRenewProhibited
Registrant Contact ID: NFVKTQY4FP7TA0W
Registrant Contact Name: EMIL ANDONOV
Tech Contact ID: NFSTGFPPDC0HFZ1Q
Tech Contact Name: EMIL ANDONOV
Name Server: DNS1.NETFLEET.COM.AU
Name Server IP: 202.191.63.6
Name Server: DNS2.NETFLEET.COM.AU
Name Server IP: 180.235.131.172
DNSSEC: unsigned
Registrant: Pacific Fund Pty Ltd
Registrant ID: ABN 75143185406
Eligibility Type: Company

>>> Last update of WHOIS database: 2018-08-07T21:25:00Z <<<



Afilias Australia Pty Ltd (Afilias), for itself and on behalf of .au Domain Administration Limited (auDA), makes the WHOIS registration data directory service (WHOIS Service) available solely for the purposes of:

(a) querying the availability of a domain name licence;

(b) identifying the holder of a domain name licence; and/or

(c) contacting the holder of a domain name licence in relation to that domain name and its use.

The WHOIS Service must not be used for any other purpose (even if that purpose is lawful), including:

(a) aggregating, collecting or compiling information from the WHOIS database, whether for personal or commercial purposes;

(b) enabling the sending of unsolicited electronic communications; and / or

(c) enabling high volume, automated, electronic processes that send queries or data to the systems of Afilias, any registrar, any domain name licence holder, or auDA.

The WHOIS Service is provided for information purposes only. By using the WHOIS Service, you agree to be bound by these terms and conditions. The WHOIS Service is operated in accordance with the auDA WHOIS Policy (available at https://www.auda.org.au/policies/index-of-published-policies/2014/2014-07/ ).
