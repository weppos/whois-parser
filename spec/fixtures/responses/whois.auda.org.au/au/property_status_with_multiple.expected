#status
  %s == ["serverRenewProhibited https://afilias.com.au/get-au/whois-status-codes#serverRenewProhibited"]
